content/en/docs/kubernetes/collector.md

```diff
@@ -0,0 +1,7 @@
+---
+title: OpenTelemetry Collector and Kubernetes
+weight: 4
+spelling: cSpell:ignore Otel
+---
+
+Coming Soon!
```

Comment:
If you don't have a content for this already, feel free to get whatever you need from here: https://medium.com/opentelemetry/deploying-the-opentelemetry-collector-on-kubernetes-2256eca569c9

Reply:
Thanks! I'll keep that in mind for future PRs.